src/components/Trends/TrendDepthToggle.jsx

```diff
@@ -0,0 +1,66 @@
+import './TrendDepthToggle.less'
+import { changeDepth, resetDepth } from '../../actions/trends'
+import { coalesce } from '../../utils'
+import { connect } from 'react-redux'
+import React from 'react'
+
+export class TrendDepthToggle extends React.Component {
+  render() {
+    const { diff, lens, resetCount, showToggle } = this.props
+    if ( showToggle ) {
+      if ( diff > 0 ) {
+        return <div className={ 'trend-depth-toggle' }>
+          <button className={ 'a-btn a-btn__link' }
+                  id={ 'trend-depth-button' }
+                  onClick={ () => {
+                    this.props.increaseDepth( diff )
+                  } }>
+            <span className={ 'plus' }></span>
+            { `View ${ diff } more ${ lens.toLowerCase() }s` }
+          </button>
+        </div>
+      }
+      return <div className={ 'trend-depth-toggle' }>
+        <button className={ 'a-btn a-btn__link' }
+                id={ 'trend-depth-button' }
+                onClick={ () => {
+                  this.props.resetDepth()
+                } }>
+          <span className={ 'minus' }></span>
+          { `View ${ resetCount } fewer ${ lens.toLowerCase() }s` }
```

Comment:
"View 13 fewer" feels awkward to me.  Was this in Matt's comp?  I would think "View only 5" or simply "View fewer" would be better

Reply:
We decided on Show More / Show Fewer in the call.